content/english/secret-messages/actividad-1.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "Actividad 1 - Output"
+date: 2019-07-25T13:24:17-07:00
+weight: 2
+draft: false
```

Comment:
I don't want these files to show up on the website yet... it would be confusing if we had an English workshop in Spanish. Please fix this by changing draft to true (and similarly everywhere):

```suggestion
draft: true
```

Reply:
I have made the changes, please advise if anything else is needed.